### DISCUSSION TITLE
Welcome to shriveling_world Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [theworldisnotflat/shriveling_world#152] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fwenner** [comment]
Dear shriveling_world team and friends,
I find the time-space compression maps really intriguing, but one thing I always asked myself is: why are all cone-tops (i.e. 'entry points' to the discontinuous transport system) at the same height? Aren't some points/cities more important than others? If a peak in the "spiky world" stands for highest accessibility, wouldn't it be possible (and desirable) to combine the shriveling-world layout with, say, a gravitational accessibility model, in the sense that the 'mass' of a region (e.g. population) also determines the height of a cone?
Best regards,
Fabian Wenner

**Reply by ghost** [reply]
Dear Fabian,
since the beginning of this endeavour I have resisted to this idea; time-space mapping, as I call it, should not be confounded with thematic cartography, hence I wanted to avoid any mix of these two types of maps. Altitude is **not** thematic in the current shriveling map, altitude is only here to convey longer time-distances according to transport mode.v
That said, curiously your observation was also introduced a few weeks ago by a colleague that works on cartography for a while Philippe Rivière, and we discussed this possibility, here are two ideas in this direction:
![dév z cones 0](https://user-images.githubusercontent.com/21110817/114238330-ba951f80-9984-11eb-9ec9-d0522997a42e.png)
And in between these two cities it exists a secondary station near Vendôme, served only by few services, so the issue is, should it be eligible to the elite club of TGV cities?
The answer could be yes, by 'pulling' Vendôme out of the _time-space abyss_ but not to the point of attaining the highest level; this could be done by considering that HST/TGV service are in fact slower, due to the lower frequencies, and we could let Vendôme emerge, while preserving the general principle that **slope =  speed ratio**
![dév z cones](https://user-images.githubusercontent.com/21110817/114238489-fa5c0700-9984-11eb-8208-85b0edc45fff.png)
Well issues remain because graphically this currently only works **if** Vendôme is exactly at the midpoint of Paris and Tours, because otherwise slopes of the slower HST would not be the same

**Reply by fwenner** [reply]
Dear Alain,
Great, thank you for sharing this background. In case of the more polycentric and federal German HSR system, there are a lot more stations in medium- and small-sized cities (= but long journeys are also slower), which had led me to the thought that not all stops can be treated in the same way. However, in France, there are also the greenfield stations like Vendôme, Haute Picardie etc. - so the solution in Fig. 1 to me seems to be a very good extension for time-space-mapping - considering two different types of trains with different speeds.

**Reply by ghost** [reply]
let's code then, I will create an issue in this sense
